text-document.rkt

```diff
@@ -342,13 +342,17 @@
      (define result
        (match decl
          [(Decl req? id left right)
+          (define new-text
+            (if (string-contains? new-name " ")
```

Comment:
That seems oddly too specific. What about other characters that need escaping as well?

Can `new-name` already contain `|`? If so, will `(string-append "|" new-name "|")` add redundant bars?

Reply:
Yes, I didn't think that carefully, I guess use `string->symbol` is enough.